README.md

```diff
@@ -6,38 +6,39 @@ This is an open-ended pair lab! It is up to you and your teammate to come up wit
 
 ## Technical Requirements
 
-Your application should be ME(A)N. This includes...
+Your application should be MEAN. This includes...
 * Using **Express** to handle HTTP requests and serve API endpoints.
 * Using **MongoDB** as your database.
 * Using **Mongoose** as your ODM.
 * Adding **Angular** as your front-end. Express should only handle your root view as well as your API routes. Angular should take care of the rest via `ui.router`.
 
 Your app should have CRUD functionality for at least one model.
 
-
 ### Bonus I
 
 Add a second model to your application, which is nested under the first.
```

Comment:
*nested* or *embedded* (to mirror the terminology of configuring mongodb document embedding)